xml/System.Diagnostics/DiagnosticListener.xml

```diff
@@ -213,10 +213,10 @@ If setup for the notification is expensive, you can call `IsEnabled` before perf
         <Parameter Name="arg2" Type="System.Object" Index="2" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="name">To be added.</param>
-        <param name="arg1">To be added.</param>
-        <param name="arg2">To be added.</param>
-        <summary>Checks if the DiagnosticListener is enabled.</summary>
+        <param name="name">Name of the event to check.</param>
+        <param name="arg1">Object that represents a context.</param>
+        <param name="arg2">Object that represents a context.</param>
+        <summary>Checks if any subscriber to the diagnostic events is interested in receiving events with this name. Subscribers indicate their interest using a delegate provided in <see cref="M:System.Diagnostics.DiagnosticListener.Subscribe%2A" />.</summary>
```

Comment:
There was a build warning, because the Subscribe methods do not have a member group item, which is why `%2A` cannot be used.

Can you please add a `<MemberGroup>` after line 300 for all the Subscribe methods, so the warning goes away?

Reply:
I added the member group. Will wait for the CI to finish and I think it'll be good to merge.